lms/djangoapps/verify_student/views.py

```diff
@@ -424,10 +424,7 @@ def get(
             'processors': processors,
             'requirements': requirements,
             'user_full_name': full_name,
-            'verification_deadline': (
-                get_default_time_display(verification_deadline)
-                if verification_deadline else ""
-            ),
+            'verification_deadline': verification_deadline or "",
```

Comment:
@nasthagiri You were right - this appeared oddly before when I passed the `None` object instead of an empty string, due to the literal `None` string evaluating as `True` in the template at https://github.com/edx/edx-platform/blob/1aff229007cbc39bbc4854a96b3d80f192c5bd26/lms/templates/verify_student/make_payment_step.underscore#L33

<img width="1224" alt="screen shot 2016-11-23 at 11 50 00 am" src="https://cloud.githubusercontent.com/assets/7373924/20571073/131cd830-b174-11e6-87fd-4b481dd3de24.png">

The fix was to add the `or ""` to this line and revert the test change back to checking for an empty string as it had before.